apps/backend/src/app/backupData/backupData.service.ts

```diff
@@ -212,6 +217,11 @@ export class BackupDataService extends PaginationService {
         },
       };
     }
+    if( backupDataOrderOptionsDto.orderBy === BackupDataOrderByOptions.SAVESET) {
+      return {
+        saveset: backupDataOrderOptionsDto.sortOrder ?? SortOrder.DESC,
+      };
+    }
```

Comment:
this shouldn't be needed, because saves is a property of the BackupDataDto and so it should work because of line 226

Reply:
Yeah you're right, thanks!